### DISCUSSION TITLE
Welcome to stdexec Discussions!

### DISCUSSION THREAD

**Discussion by ericniebler** [NVIDIA/stdexec#654] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gamecentric** [comment]
Despite claims that commit https://github.com/NVIDIA/stdexec/commit/b62ecdfe84e0b59411a9ab9b47d5f4fe539e8412 implements [P3149R11](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p3149r11.htm), it seems to actually implement more like R7 or a prior revision, since it's missing a key change introduced in R8. In particular, the code still relies on the user-visibile scope_association concept that has been removed from the revision integrated in the latest draft of C++26. I am working on full implementation of R11, is there any interest for a PR?

**Reply by villevoutilainen** [reply]
> Despite claims that commit [b62ecdf](https://github.com/NVIDIA/stdexec/commit/b62ecdfe84e0b59411a9ab9b47d5f4fe539e8412) implements [P3149R11](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p3149r11.htm), it seems to actually implement more like R7 or a prior revision, since it's missing a key change introduced in R8. In particular, the code still relies on the user-visibile scope_association concept that has been removed from the revision integrated in the latest draft of C++26. I am working on full implementation of R11, is there any interest for a PR?

This paper https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p3815r1.html brought back the user-visible scope_association concept, and was adopted into C++26.

**Reply by gamecentric** [reply]
I see. Never mind, then. Thanks for pointing me to right paper, as I am having a hard time tracking the "most recent" specification.

**Reply by ericniebler** [reply]
https://eel.is/c++draft/exec always represents the latest, and it now has the updates from the recent committee meeting.
